net-wireless/hostapd/hostapd-9999.ebuild

```diff
@@ -205,35 +202,35 @@ src_compile() {
 }
 
 src_install() {
-	insinto /etc/${PN}
-	doins ${PN}.{conf,accept,deny,eap_user,radius_clients,sim_db,wpa_psk}
+	insinto "/etc/${PN}"
+	doins "${PN}".{conf,accept,deny,eap_user,radius_clients,sim_db,wpa_psk}
 
-	fperms -R 600 /etc/${PN}
+	fperms -R 600 "/etc/${PN}"
 
-	dosbin ${PN}
-	dobin ${PN}_cli
+	dosbin "${PN}"
+	dobin "${PN}_cli"
 
 	if ! use internal-tls; then
 		dobin nt_password_hash hlr_auc_gw
 	fi
 
-	newinitd "${WORKDIR}/${EXTRAS_NAME}"/${PN}-init.d ${PN}
-	newconfd "${WORKDIR}/${EXTRAS_NAME}"/${PN}-conf.d ${PN}
-	systemd_dounit "${WORKDIR}/${EXTRAS_NAME}"/${PN}.service
+	newinitd "${FILESKDIR}/${PN}-init.d" "${PN}"
```

Comment:
Typo. Also, PN doesn't need quoting, nor do bare paths unless they contain a variable with a space. I prefer not to quote those.